Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py

```diff
@@ -1,43 +1,97 @@
+import demistomock as demisto
 from CommonServerPython import *
 import json
+import re
 
+ERROR_TEMPLATE = 'ERROR: PreprocessEmail - {function_name}: {reason}'
+QUOTE_MARKERS = ['<div class="gmail_quote">',
+                 '<hr tabindex="-1" style="display:inline-block; width:98%"><div id="divRplyFwdMsg"']
 
-def add_entries(single_reply, email_related_incident):
+
+def create_email_html(email_html='', entry_id_list=[]):
+    """Modify the email's html body to use entry IDs instead of CIDs and remove the original message body if exists.
+    Args:
+        email_html (str): The attachments of the email.
+        entry_id_list (list): The files entry ids list.
+    Returns:
+        str. Email Html.
+    """
+
+    # Removing the conversation's history
+    for marker in QUOTE_MARKERS:
+        index = email_html.find(marker)
+        if index != -1:
+            email_html = f'{email_html[:index]}</body></html>'
+
+    # Replacing the images' sources from
+    for entry_id in entry_id_list:
```

Comment:
how does entry_id look like? what is the first object? what is the second?
split it into different variables: `for image_name, image_path in entry_id_list:` (I guessed the meanings)